src/univariate/continuous/inversegaussian.jl

```diff
@@ -161,3 +161,37 @@ function rand(rng::AbstractRNG, d::InverseGaussian)
     u = rand(rng)
     u >= p1 ? μ^2 / x1 : x1
 end
+
+#### Fit model
+
+struct InverseGaussianStats <: SufficientStats
```

Comment:
add documentation for `InverseGaussianStats`

Reply:
to add documentation, do:
"""
Sufficient statistics for the inference of an inverse Gaussian
"""
struct InverseGaussianStats <: SufficientStats